renovate.json

```diff
@@ -1,5 +1,10 @@
 {
   "extends": [
     "config:base"
-  ]
+  ],
+  "packageRules": [{
+    "packagePatterns": [".*"],
+    "groupName": "packages",
+    "schedule": ["after 9pm on sunday"]
+  }]
```

Comment:
I think the only line you really need is the one with `schedule`. The file could be simplified to:
```
{
  "extends": [
    "config:base"
  ],
  "schedule": "after 9pm on sunday"
}
```

Reply:
`packagePatterns` + `groupName` is what causes grouping of package updates. Without this, we'll get dozens of update PRs at the same time

Reply:
Aha, missed that part.